src/status_im/contexts/wallet/swap/events.cljs

```diff
@@ -6,7 +6,8 @@
 
 (rf/reg-event-fx :wallet.swap/start
  (fn [{:keys [_db]}]
-   {:fx [[:dispatch [:open-modal :screen/wallet.swap-select-asset-to-pay]]]}))
+   {:fx [[:dispatch [:open-modal :screen/wallet.swap-select-asset-to-pay]]
+         [:dispatch [:wallet.tokens/get-token-list]]]}))
```

Comment:
Probably we want to preload the token list at login, we will need it on other flows, for example on #20927, which is not dependant on swaps feature